dashboard/test/models/pd/application/teacher2021_application_test.rb

```diff
@@ -744,15 +727,6 @@ class Teacher2021ApplicationTest < ActiveSupport::TestCase
       )
     end
 
-    test 'autoscore is idempotent' do
-      application = create :pd_teacher2021_application, regional_partner: nil
-      application.update(response_scores: {regional_partner_name: YES}.to_json)
-
-      application.auto_score!
-
-      assert_equal YES, JSON.parse(application.response_scores)['regional_partner_name']
-    end
```

Comment:
Interesting offline conversation about removing this test - seems there may no longer be a reason to keep the autoscore function from updating scores, but want to continue that conversation later.